### PR TITLE
Adding ovms info.

### PR DESCRIPTION
**Pull request by rasapala**

### CODE REVIEW

File: store/data/projects.yaml [hunk 1]
```diff
@@ -1036,3 +1036,28 @@
...
+
+    - Simultanous serving of any model trained in a framework that is supported by OpenVINO
+
+    - The server implements gRPC and REST API framework with data serialization and deserialization using TensorFlow Serving API
+
+    - Uses OpenVINO™ as the inference execution provider
+
+    - Supports different file systems: local (e.g. NFS), Google Cloud Storage (GCS), Amazon S3, Minio or Azure Blob Storage
+
+    - OVMS is now implemented in C++ and provides scalability, low latency and high perormance
```
Comment (atobiszei) [inline store/data/projects.yaml]: I think this can be ommitted

Comment (atobiszei) [reply store/data/projects.yaml]: You already mention in 1st sentence sth similar

File: store/data/projects.yaml [hunk 2]
```diff
@@ -1036,3 +1036,28 @@
   introLogoVisible: true
   gitHubRepoName: treeverse/lakeFS
   youTubeVideoId: xThorxDzmrw
```
Comment (atobiszei) [inline store/data/projects.yaml:1038]: shouldn't this be emptied or changed to the official one?

Comment (rasapala) [reply store/data/projects.yaml:1038]: video url for OVMS is in the last line: youTubeVideoId: AfytPrAVdfc

Comment (atobiszei) [reply store/data/projects.yaml:1038]: So shouldn't we remove this line?

Comment (rasapala) [reply store/data/projects.yaml:1038]: No this is part of previous project not ovms.

**Review by atobiszei — APPROVED** [review]

**Pull request merged**